intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+package com.jetbrains.plugin.structure.ktor
+
+import com.jetbrains.plugin.structure.base.plugin.Plugin
+import com.jetbrains.plugin.structure.base.plugin.PluginIcon
+
+data class KtorFeature(
+  override val pluginName: String? = null,
+  override val description: String? = null,
+  override var vendor: String? = null,
+  override var vendorEmail: String? = null,
+  override var vendorUrl: String? = null,
+  override val icons: List<PluginIcon> = emptyList(),
```

Comment:
Why is it a list here but not single icon?

Reply:
it will be a list with only one icon